components/collection/activity/events/Events.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div ref="eventsContainer">
+    <div v-if="events.length > 0">
+      <div v-if="desktop" class="columns is-size-7 has-text-grey">
+        <div class="column">
+          <span>{{ $t('activity.event.item') }}</span>
+        </div>
+        <div class="column">
+          <span>{{ $t('activity.event.event') }}</span>
+        </div>
+        <div class="column">
+          <span>{{ $t('activity.event.amount') }}</span>
+        </div>
+        <div class="column">
+          <span>{{ $t('activity.event.from') }}</span>
+        </div>
+        <div class="column">
+          <span>{{ $t('activity.event.to') }}</span>
+        </div>
+        <div class="column">
+          <span>{{ $t('activity.event.time') }}</span>
+        </div>
+      </div>
+      <EventRow
+        v-for="event in displayedEvents"
+        :key="event.timestamp"
+        :variant="variant"
+        :event="event" />
+      <div ref="sentinel" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  InteractionWithNFT,
+  Offer,
+  OfferInteraction,
+} from '@/components/collection/utils/types'
+import EventRow from './EventRow.vue'
+import { useIntersectionObserver } from '@vueuse/core'
+import { Interaction } from '@kodadot1/minimark'
+import { is } from '@/composables/useReplaceUrl'
```

Comment:
```suggestion
import { is } from '@/composables/useReplaceUrl'
```

`useReplaceUrl` is [auto imported](https://nuxt.com/docs/guide/directory-structure/composables#composables-directory) here